catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/XmlInputTransformer.java

```diff
@@ -288,10 +293,10 @@ public String getName() {
 
     @Override
     public AttributeDescriptor getAttributeDescriptor(String attributeName) {
-        return attributeDescriptors.stream()
```

Comment:
Hm, there might be more idiomatic solutions with Java 8. I feel like I've seen null handling or contains checks in the streaming pattern before.